Assets/Scripts/Commanders/BombBinderCommander.cs

```diff
@@ -225,8 +240,39 @@ private void MoveUpOnPage()
         _currentSelectableIndex = oldSelectableIndex;
     }
 
-    private IEnumerator SelectOnPage()
+    private IEnumerator SelectOnPage(int index = 0)
     {
+        if (index > 0)
+        {
+            if ( (_currentSelectables == null) || (index > _currentSelectables.Length) )
+            {
+                yield break;
+            }
+
+            int i = 0;
+            MonoBehaviour newSelectable = null;
+            for (_currentSelectableIndex = 0; _currentSelectableIndex < _currentSelectables.Length; ++_currentSelectableIndex)
+            {
+                Debug.Log(string.Format("[bmn] Looking for index {0}", _currentSelectableIndex));
```

Comment:
The user-specific parts of these log messages were something I intended to take out before PRing but apparently missed some. I'll go over it properly and update.